Source/GoBot/ViewDatabase.swift

```diff
@@ -2782,6 +2913,20 @@ class ViewDatabase {
         ))
     }
     
+    func roomID(of aliasURL: String) throws -> Int64? {
```

Comment:
Could you add a doc comment for this function? Especially to describe the behavior of when it will throw vs. returning nil.